lib/rubocop/cop/style/self_assignment.rb

```diff
@@ -82,13 +78,11 @@ def autocorrect(corrector, node)
         end
 
         def autocorrect_send_node(corrector, node, rhs)
-          _receiver, method_name, args = *rhs
```

Comment:
I guess this worked by accident before. `args` gets assigned the _first_ argument to the rhs `SendNode`.

Reply:
`args` was already the first argument here, any node children after 3 were just ignored.